### PR TITLE
fix(workflowengine): Use correct event names to make search for groups work again

### LINKED ISSUE #39179: Groups missing in Flow app
### Steps to reproduce
1. I have 32 groups only 21 are displayed in the drop down

### Expected behaviour
All groups should be displayed

### Actual behaviour
only 21 are displayed in the drop down

**Nextcloud version:** (see Nextcloud admin page)
26.0.2

### PR DESCRIPTION
**Pull request by susnux**
* Resolves: https://github.com/nextcloud/server/issues/39179

## Summary

Adjust the event name to make searching for groups work again, this way users can select more than the initial 20 groups.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/workflowengine/src/components/Checks/RequestUserGroup.vue [hunk 1]
```diff
@@ -22,21 +22,26 @@
...
+			:aria-label-listbox="t('workflowengine', 'Groups')"
+			:clearable="false"
 			:loading="status.isLoading && groups.length === 0"
+			:placeholder="t('workflowengine', 'Type to search for group …')"
 			:options="groups"
-			:clearable="false"
+			:value="currentValue"
 			label="displayname"
-			@search-change="searchAsync"
+			@search="searchAsync"
```
Comment (nickvergessen) [inline apps/workflowengine/src/components/Checks/RequestUserGroup.vue:33]: I just fixed that in another app as well :see_no_evil:

**Review by nickvergessen — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**